sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/AnySupport.scala

```diff
@@ -379,7 +382,7 @@ class AnySupport(
 
       case null =>
         throw SerializationException(
-          s"Don't know how to serialize object of type null. Try passing a protobuf, using a primitive type, or using a type annotated with @Jsonable.")
+          s"Don't know how to serialize object of type null. Try passing a protobuf, using a primitive type, or defining a codec for this type.")
```

Comment:
Except that users cant "define a codec" since it is internal API, so not a great error message? Also this is if the _value_ was null, would fail regardless, so just a NPE could suffice?

Reply:
Will fix.